dlt/load/load.py

```diff
@@ -68,6 +68,7 @@ def __init__(
         config: LoaderConfiguration = config.value,
         initial_client_config: DestinationClientConfiguration = config.value,
         initial_staging_client_config: DestinationClientConfiguration = config.value,
+        refresh: Optional[TRefreshMode] = None,
```

Comment:
why we need this? IMO it is enough if we keep dropped and truncated tables in package state and just execute the changes

Reply:
Yes, keeping truncated tables would be cleaner. This doesn't need to know about refresh